### ISSUE TITLE
Ohh...WinUI3 is really dead! - When can we expect the announcement?

### ISSUE THREAD

**Issue by softworkz** [microsoft/microsoft-ui-xaml#9414]
Yesterday I made a post  ([Great News: All Bugs Will be Fixed In the Year 2041](https://github.com/microsoft/microsoft-ui-xaml/issues/9410)) which was meant to be a somewhat funny complaint about the lack of activity in terms of development progress and dealing with user issues submitted here.

Then I made a [comparison of activity between WinUI3 and MAUI](https://github.com/microsoft/microsoft-ui-xaml/issues/9410#issuecomment-1985068849) and those figures are quite revealing when thinking about it:

How can it be possible that MS is investing 20 times more effort into MAUI than into WinUI3 - the new and designated primary UI technology for Windows desktop? 
Simple answer: It can't. It can only be possible in case when WinUI3 is no longer the "designated primary UI technology for Windows desktop".

Now I'm shocked to realize that it's way more serious than I thought as so many details suddenly fit together and make sense when being viewed in the light of that premise.

### Smaller Bits

- No substantial official statements or responses are being made on GitHub anymore
- The [Product Board](https://portal.productboard.com/winappsdk/1-windows-app-sdk/tabs/2-planned) is outdated and no longer maintained
- The hard drop of WinUI2 support wouldn't have been needed when still working in WinUI3, because large parts of WinUI2/3 code are identical, so certain fixes to WinUI3 could have easily been applied to WinUI2. 
- The whole architecture behind this is ridiculously complex, all the projections, WinRT, the insane build system for the WinUI3 code, etc.
  - Experience is so inferior compared to .net based implementations - for both, framework consumers and developers
  - Inability to ship a visual designer (surely also due to the aforementioned)
  - limitations in the composition architecture  (no external content overlay)
- Adoption from component vendors hasn't been great

### Xbox Development

- No announcement has been made regarding the future of Xbox development
  - UWP is dead
  - WinUI2 declared as unsupported from one day to another
  - WinUI3 doesn't work on Xbox
  => but Xbox is not dead. So the must be something for Xbox which is not WinUI3

### Component Vendors

DevExpress are offering a few free components but these haven't been updated since mid-2022.
Their blog entries about WinUI3 stopped in mid-2022 as well (https://search.devexpress.com/?m=Web&q=winui3)

They probably know already about the end of WinUI3.

### Timely Coincidence

Wasn't it at the same time (2022) that WinUI3 development had started to be driven down?

### XAML Islands as a WinUI3 Exit Strategy

When looking at the changes that were made during the past year, then there's only one area where significant work has been put into: XAML Islands. Even though it had been requested by users earlier, it's not a most-wanted feature, but still, MS picked exactly that part as the one major subject of work.
XAML Islands allows to integrate WinUI3 content as "islands" when using other UI frameworks. Why is this so important? Who needs this other than a few with very specific use cases? 
When customers want to modernize their applications, who would start by implementing something in WinUI3 which is then shown like a control/window within a WPF or WinForms app? Normally, such migrations would rather go the other way round: starting with a new application framework and subsequently migrating legacy components. But that's what is not being worked: There's nothing like a hWnd-Panel (which had also been requested by users). 

Bottom line: When establishing a new framework for app development, then it's usually crucial to provide ways for developers to integrate legacy components into the new framewrok to  allow subsequent migration, but not the other way round. Provisioning for the other direction only (and also as the only major change) can only mean that this is the end of the road for WinUI3, and XAML Islands are just implemented tto soften the impact for affected developers enabling a life-after-death existence for WinUI3 components.

**Comment by softworkz** [comment]
# Component Vendor Support for WinUI3

## Telerik

![image](https://github.com/microsoft/microsoft-ui-xaml/assets/4985349/7f4a62a8-e68e-43fa-b1f2-9474b353ee62)

The two "under review" haven't been updated since about a year. 
All accepted features are set to "Unplanned"

Ref: https://feedback.telerik.com/winui


## Syncfusion

The WinUI roadmap for 2024 includes only improvements to shared components (i.e. improvements are made for multiple platforms) but nothing specific to WinUI3 controls.

Ref: https://www.syncfusion.com/products/roadmap/winui-controls


## DevExpress

They are explicitly stating that their WinUI3 components are dead:

![image](https://github.com/microsoft/microsoft-ui-xaml/assets/4985349/13650870-1b25-42a7-bd8c-f9b4e097f314)

Ref: https://supportcenter.devexpress.com/ticket/details/t1206625/winui-3-the-roadmap-in-2024

**Comment by softworkz** [comment]
## Consequences

If this turns out to be true, this would be a severe incident:

### 1. You owe me two days of work

For creating the Pull Requests, since I didn't know that they'll be never even looked at due the product being abandonded.

Same goes for the effort that so many people have taken in order to submit bug reports or create design proposals.
You have been watching everybody doing it, even though you knew that they will never be worked on and fixed.


### 2. Have Microsoft Representatives been Lying to the Developer Community?

This needs to be evaluated. Behaving as if all would be fine and the product would be continued is quite close already.

And I recall statements like "all developers are working on WinUI3 now" which I think were made after 2022 and after stopping development (largely),

When the UWP bugs were all closed, it was promised that issues will be better handled in the future and it won't happen again.
IIRC, this was also after 2022.
